app/Http/Controllers/Secretariat/UserController.php

```diff
@@ -314,6 +314,7 @@ public function updateTenantUntil(Request $request, User $user)
         $validator->validate();
 
         $date = min(Carbon::parse($request->tenant_until), Carbon::now()->addMonths(6));
+        $user->addRole(Role::get(Role::TENANT));
```

Comment:
this might add multiple Tenant roles for the same user (tenants have to extend their status regularly)

Reply:
I see... I put it into an `if` then.